### DISCUSSION TITLE
ممکنه وقتی که متد wait_for درباره پیام ها درحال اجرا هست ...

### DISCUSSION THREAD

**Discussion by daradege** [kianahmadian/python-bale-bot#126] [Ideas] [answered]
سلام
ممکنه وقتی که متد wait_for درباره پیام ها درحال اجرا هست 
وقتی که callback جدیدی ارسال میشه اون متد غیر فعال شه؟
اگه نه میتونین توی آپدیت بعدی اضافه کنیدش؟

**Comment by kian-ahmadian** [comment]
سلام عزیز وقتتون بخیر
متوجه منظورتون نشدم لطفا بهتر منظورتون و بیان کنید

**Comment by daradege** [comment, marked as answer]
سلام فیکس شد
برای افرادی که سوالشون مشابه بود : با یک لیست که داخلش افراد بودن درست شد
